html-practice/hackernews.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<style type="text/css">
+			body {
+				font-family: Arial;
+				font-size: 0.85em;
+			}
+			main {
+				background-color: #f6f6ef;
+				margin: 0 100px;
+			}
+			header {
+				background-color: #ff6600;
+			}
+			footer {
+				border-top: 2px solid #ff6600;
+				text-align: center;
+				text-decoration: none;
+			}
+			#logo {
+				margin: 2px;
+			}
+			.title {
+				font-weight: bold;
+			}
+			nav {
+				display: inline-block;
+			}
+			a {
+				color: #000;
+				text-decoration: none;
+			}
+			nav a::after {
+				content: ' |';
+			}
+			nav a:last-child::after {
+				content: '';
+			}
+			a.login-button {
+				float: right;
+			}
+			.applications-link {
+				margin: 20px;
+				display: block;
+				color: grey;
+			}
+			.footer-nav {
+				margin-bottom: 20px;
+			}
+			.paragraph-link {
+				color: grey;
+				letter-spacing: 1px;
+				font-size: 0.85em;
+			}
+			ol p {
+				color: grey;
+				letter-spacing: 1px;
+				font-size: 0.80em;
+				margin: 2.5px 0px 5px;
+			}
+			ol li {
+				letter-spacing: 1px;
+				font-size: 0.95em;
+				color: grey;
+			}
+			ol li span {
+				letter-spacing: 1px;
+				font-size: 0.95em;
+				color: black;
+			}
+		</style>
+	</head>
+	<body>
+		<main>
+			<header>
+				<img src="logo.png" id="logo">
+				<span class="title">
+					Hacker News
+				</span>
+				<nav>
+					<a href="#">new</a>
+					<a href="#">comments</a>
+					<a href="#">show</a>
+					<a href="#">ask</a>
+					<a href="#">jobs</a>
+					<a href="#">submit</a>
+				</nav>
+				<a class="login-button">
+					login
+				</a>
+			</header>
+			<section class="content">
+				<ol> 
+					<li> <span>J.C.R Licklider on the History of Personal Workstations (1986) [video] </span><font class="paragraph-link"> (youtube.com)</font></li>
+						<p> 98 points by dang 6 hours ago | 16 comments </p>
```

Comment:
Replace those `<font>` tags with `<span>`s.

`<font>` has been deprecated since HTML 4.01 came out in 1999. So... Don't ever use it. :-)